tests/dragonfly/seeder/script-genlib.lua

```diff
@@ -62,7 +92,15 @@ function LG_funcs.add_set(key, keys)
         end
         redis.apcall('SDIFFSTORE', key, keys[i1], keys[i2])
     else
-        local elements = dragonfly.randstr(LG_funcs.esize, LG_funcs.csize)
+        local elements
+        if huge_entry() then
+            -- Hard coded 10 here, meaning up to 10 huge entries per set
```

Comment:
//TODO so I don't forget to fix it. replace `10` with `LG_funcs.csize()`

Reply:
please fix :)